modules/phase_field/include/action/MultiAuxVariablesAction.h

```diff
@@ -0,0 +1,34 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef MULTIAUXVARIABLESACTION_H
+#define MULTIAUXVARIABLESACTION_H
+
+#include "InputParameters.h"
+#include "Action.h"
+
+/**
+ * Automatically generates all auxvariables given vectors telling it the names
+ * and how many to create
+ */
+class MultiAuxVariablesAction: public Action
```

Comment:
I'm not sure if this is a normal thing, or some sort of glitch, but my changes are not showing up on the Conversation tab. I have to click "View full changes" in order to see the changes I've made. It now says `#include "AddAuxVariableAction"`, instead of what is shown here. Is there another change here that I should make that I'm not seeing?